utils.php

```diff
@@ -66,4 +66,40 @@ public function parse_string_for_csv($raw_values, $delimiter = ',', $enclosure =
 	}
 }
 
+class database_utils {
+    /**
+     * Get all questions having the selected topic and difficulty
+     */
+    public function get_questions($topic, $difficulty, $categoryid) {
+        global $DB;
+
+        global $sql_questionids_anytopic_anydifficulty;
+        global $sql_questionids_anydifficulty;
+        global $sql_questionids_anytopic;
+        global $sql_questionids;
```

Comment:
I think you could move the queries to this file. I don't think there's any point in keeping them in the `consts` file if they are not used in another context.